crates/kitsune_p2p/proxy/src/inner_listen.rs

```diff
@@ -391,14 +414,15 @@ impl InternalHandler for InnerListen {
         .into())
     }
 
+    #[tracing::instrument(skip(self))]
     fn handle_register_proxy_to(
         &mut self,
         proxy_url: ProxyUrl,
         base_url: url2::Url2,
     ) -> InternalHandlerResult<()> {
         // expire ProxyTo entries at double the proxy keepalive timeframe.
         let expires_at = std::time::Instant::now()
-            .checked_add(std::time::Duration::from_millis(PROXY_KEEPALIVE_MS * 2))
+            .checked_add(std::time::Duration::from_millis(PROXY_KEEPALIVE_MS * 3))
```

Comment:
can you adjust the comment above to match?

Reply:
I think it's the same const. Are you happy with 3x? or should I put it back to 2x. I might make that multiplier a const too.